front/app/components/admin/PostManager/components/FilterSidebar/statuses/FilterSidebarStatuses.tsx

```diff
@@ -49,10 +54,17 @@ const FilterSidebarStatuses = ({
       : '/admin/settings/proposals/statuses';
 
   return (
-    <Menu secondary={true} vertical={true} fluid={true}>
-      <Menu.Item onClick={clearFilter} active={!selectedStatus}>
+    <Box display="flex" flexDirection="column">
+      <Button
+        onClick={clearFilter}
+        bgColor={!selectedStatus ? colors.grey200 : 'transparent'}
+        buttonStyle="text"
+        justify="left"
+        bgHoverColor={colors.background}
+        fontSize={`${fontSizes.s}px`}
```

Comment:
Uses the same button styles as the single status filters, but was a bit hard to extract (+ I would use radio inputs rather than buttons if I'd do any refactoring).